development/docker-compose.yml

```
@@ -20,7 +18,7 @@ services:
       nautobot:
         condition: "service_healthy"
   nautobot:
-    image: "ghcr.io/nautobot/nautobot-dev:${NAUTOBOT_VER:-1.6}-py${PYTHON_VER:-3.8}"
+    image: "ghcr.io/nautobot/nautobot-dev:${NAUTOBOT_VER:-2.0}"
```

```suggestion
    image: "ghcr.io/nautobot/nautobot-dev:${NAUTOBOT_VER:-1.6}"
```

Copy pasta from the 2.x changes